src/main/resources/vm/script/datamanagement/downloadFunctions.vm

```diff
@@ -97,22 +97,29 @@ export -f downloadGirderFile
 
 #This function identifies the gfal path and extracts the basename of the directory to be mounted, and creates a directory with the exact name on $PWD of the node.
 #This directory gets mounted with the corresponding directory on the SE.
-#Only directories must be mounted using gfal (in case of mounting of a file, there will be an empty mount and it risks the failure of the application) - e.g-$SRM_HOST/home/biomed/user/....  
 
+
+check_mount='$(test -z $(for file in *; do findmnt -t fuse.gfalFS -lo Target -n -T $(realpath ${file}); done) && echo 1 || echo 0)'
```

Comment:
i'm not sure what it does.
Does it check for every file/folder in the pwd if it is a gfal mount ?
So if there is at least 1 gfal mount in pwd it returns 0, else 1 ?
If that is the case, could you add a comment with that ?